### PR TITLE
Issue 4648 pull down url

### LINKED ISSUE #4648: bounty/new -  it doesnt pull down the desc when you paste the URL in there
**Describe the bug**

 it doesnt pull down the desc when you paste the URL in top of form

**To Reproduce**

paste url in gitcoin.co/new

**Expected behavior**

the summary of the issue should pull down without pushing 'sync with github'

if you go to https://gitcoin.co/bounty/new?source?=[url], it should pull the github info from [url] upon pageload

**Additional context**

this is the issue that @PixelantDesign @coderberry reported from their pairing session last week

### PR DESCRIPTION
**Pull request by iamonuwa**
##### Description

Fix loading data for pasted github url.

##### Refers/Fixes

Fixes #4648 

##### Testing
This is a frontend issue. There is no test. To test on your end, gitcoin.co/bounty/new?source=<github_issue_url> will now load the description and title of the issue.

### CODE REVIEW

**Comment by danlipert** [comment]
@iamonuwa can you post a screenshot or video demonstrating the fix? thanks!

File: app/assets/v2/js/shared.js [hunk 1]
```diff
@@ -656,21 +656,21 @@ const showChoices = (choice_id, selector_id, choices) => {
   });
 };
 
-var retrieveIssueDetails = function() {
-  var ele = $('input[name=issueURL]');
-  var target_eles = {
+const retrieveIssueDetails = function() {
+  let ele = $('input[name=issueURL]');
+  let target_eles = {
```
Comment (danlipert) [inline app/assets/v2/js/shared.js]: Please try not to make changes to the codebase with automated tools that fix style, etc. - it makes it more difficult to review the code. These changes aren't too bad but as a general rule, minimize your changes to the functionality described in the PR/issue.

**Comment by iamonuwa** [comment]
> @iamonuwa can you post a screenshot or video demonstrating the fix? thanks!

Here you go https://www.dropbox.com/s/sjlprqde86iun9j/Gitcoin%20pull%20data.mp4?dl=0

**Comment by danlipert** [comment]
@octavioamu please review

**Pull request merged**